src/Change.php

```diff
@@ -818,8 +818,8 @@ public static function showListForItem(CommonDBTM $item, $withtemplate = 0)
                 '_add_fromitem',
                 __('New change for this item...'),
                 [
-                    '_from_itemtype' => $item->getType(),
-                    '_from_items_id' => $item->getID(),
+                    'itemtype' => $item->getType(),
```

Comment:
```suggestion
                    'itemtype' => $item::getType(),
```
You should be able to use `$item::class` as well and be more performant.
https://php.watch/versions/8.0/class-constant-on-objects
As far as I can tell, `getType` has no purpose anymore. It is a relic from before PHP had the `static` keyword.

Reply:
done